ibm_db2/datadog_checks/ibm_db2/ibm_db2.py

```diff
@@ -65,6 +75,21 @@ def check(self, instance):
         self.query_table_space()
         self.query_transaction_log()
         self.query_custom()
+        self.collect_metadata()
+
+    def collect_metadata(self):
+        try:
+            version = get_version(self._conn)
+        except Exception as e:
+            self.log.error("Error getting version: {}".format(str(e)))
+            return
+
+        if version is not None:
```

Comment:
to handle the 'False' case
```suggestion
        if version:
```